core/server/common/src/main/java/alluxio/master/journal/raft/RaftJournalSystem.java

```diff
@@ -206,8 +208,8 @@ private synchronized void initServer() {
     mStateMachine = new JournalStateMachine(mJournals);
     mServer = CopycatServer.builder(getLocalAddress(mConf))
         .withStorage(storage)
-        .withElectionTimeout(Duration.ofMillis(mConf.getElectionTimeoutMs()))
         .withHeartbeatInterval(Duration.ofMillis(mConf.getHeartbeatIntervalMs()))
+        .withElectionTimeout(Duration.ofMillis(mConf.getElectionTimeoutMs()))
```

Comment:
I think this could fail if someone wants to set (heartbeat timeout, election timeout) to (10s, 30s). Could we add a test for this?

Reply:
The copycat server is really strange.  `electionTimeout=100ms, heartbeatTimeout=20ms` failed no master we set heartbeat interval or election timeout first.
but `electionTimeout=120ms, heartbeatTimeout=20ms` succeed when setting the heartbeat interval first

Reply:
I changed back the sequence (`withElection` first and `withHeartbeat` later).
As long as election timeout is bigger than the default heartbeat interval `250ms`, there will be no issue with copycat.

```
      conf.put(PropertyKey.MASTER_EMBEDDED_JOURNAL_ELECTION_TIMEOUT, "260ms");
      conf.put(PropertyKey.MASTER_EMBEDDED_JOURNAL_HEARTBEAT_INTERVAL, "50ms");
```

Reply:
What would it take to make it work even when the election timeout is smaller than `250ms`?